pages/blog/index.tsx

```diff
@@ -23,7 +23,7 @@ export default function Blog(props: InferGetStaticPropsType<typeof getStaticProp
                             </div>
                             <h2 className='text-lg font-bold sm:text-xl md:text-2xl'>
                                 <Link href={`/blog/${page.slug}`}>
-                                    <a>{page.title}</a>
+                                    {page.title}
                                 </Link>
                             </h2>
                             <p className='text-sm text-gray-500'>{page.teaser}…</p>
```

Comment:
```suggestion
                            <p className='text-sm'>{page.teaser}…</p>
```
make blog teaser text white again

Reply:
and please make the next line
```
<p className='pt-2 text-xs font-medium text-gray-500'>
```
can't add a suggestion there.
![Screenshot_20230916_152919](https://github.com/tidev/tidev-www/assets/4334997/80ee0bd8-558d-410d-9350-1c0bb38c65c1)